components/well-bold/well-bold-en.html

```diff
@@ -1,37 +1,20 @@
 ---
-breadcrumbs: [
+breadcrumb: [
 	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/home.html" }
 ]
-dateModified: "2021-05-25"
-description: "Documentation and working example for the well bold - provisional CSS"
+dateModified: "2022-04-28"
+description: "Working example for the well bold"
 language: "en"
 altLangPage: "well-bold-fr.html"
-title: "Well bold - Provisional CSS"
+title: "Well bold"
 ---
-{% include alert-provisional.html siteroot="../../" %}
 
 <span class="wb-prettify all-pre hide"></span>
 
-<dl class="horizontal">
-	<dt>Status</dt>
-	<dd>Provisional</dd>
-	<dt>Type</dt>
-	<dd>Utility</dd>
-	<dt>Last review</dt>
-	<dd>2021-05-25</dd>
-	<dt>Conforming to</dt>
-	<dd>n/a</dd>
-	<dt>Nom technique</dt>
-	<dd><code>well-bold</code></dd>
-</dl>
-
-<h2>Purpose</h2>
-<p>This apply font weight bold to evey text element inside a well component and generates normal font weight when the element <code>strong</code> is applied.</p>
-
 <h2>Working example</h2>
 
-<h3>Well bold (provisional v1.0)</h3>
-<section class="provisional well well-bold">
+<h3>Well bold (v1.0)</h3>
+<section class="well well-bold">
 	<div class="row">
 		<section class="col-md-6">
```

Comment:
Une section devrais contenir un titre. Je suggère de convertir la `section` en `div`.
```suggestion
		<div class="col-md-6">
```

Reply:
Changed.Thanks!